charts/karpenter/templates/configmap-grafana-dashboards-karpenter-capacity-dashboard.yaml

```diff
@@ -0,0 +1,1459 @@
+{{- if .Values.grafana.dashboards.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: grafana-dashboard-karpenter-capacity-dashboard
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "karpenter.labels" . | nindent 4 }}
+  {{- with .Values.grafana.dashboards.additionalLabels }}
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- with .Values.additionalAnnotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+data:
+  # https://github.com/aws/karpenter-provider-aws/blob/main/website/content/en/docs/getting-started/getting-started-with-karpenter/karpenter-capacity-dashboard.json
+  karpenter-capacity-dashboard.json: |
```

Comment:
Can we consider storing the JSON in a separate JSON file and [then having Helm inject the file into the ConfigMap](https://helm.sh/docs/chart_template_guide/accessing_files/#basic-example)? IMO, that'll make each of these files easier to maintain.

Reply:
This would generally be done via a `range` so a single template could handle all of the dashboards in a folder but keep them as separate ConfigMap resources.

Reply:
ok yeah, I should have thought about that! will make change